src/main/java/org/mifos/connector/channel/camel/routes/ChannelRouteBuilder.java

```diff
@@ -247,6 +247,28 @@ public void configure() {
                     exchange.getIn().setBody(response.toString());
                 });
 
+        /*
+         * Tester endpoint for starting mpesa workflow
+         */
+        from("rest:POST:/channel/mpesa/transaction")
+                .id("mpesa-payment-request")
+                .log(LoggingLevel.INFO, "## CHANNEL -> MPESA transaction request")
+                .to("bean-validator:request")
+                .process(exchange -> {
+
+                    Map<String, Object> extraVariables = new HashMap<>();
+                    extraVariables.put("initiator", "PAYEE");
+                    extraVariables.put("initiatorType", "BUSINESS");
+                    extraVariables.put("scenario", "MPESA");
+
+                    String transactionId = zeebeProcessStarter.startZeebeWorkflow("Process_0zpkte0",
```

Comment:
checkout [line 242](https://github.com/openMF/ph-ee-connector-channel/blob/master/src/main/java/org/mifos/connector/channel/camel/routes/ChannelRouteBuilder.java#L242) in the same file, I used it as the reference.

Reply:
Line 242 has tenantSpecificBpmn which is a fully qualified name concatenated with tenant name.

Reply:
Ideally this would work but not a good practice. Best to read the workflow name from application properties and make necessary changes instead of hardcoding it.